packages/react/src/components/Toggle/Toggle.Skeleton.js

```diff
@@ -26,17 +27,22 @@ export default class ToggleSkeleton extends React.Component {
      */
     labelText: PropTypes.string,
     ['aria-label']: PropTypes.string.isRequired,
+
+    /**
+     * Specify an optional className to add to the form item wrapper.
+     */
+    className: PropTypes.string,
   };
 
   static defaultProps = {
     ['aria-label']: 'Toggle is loading',
   };
 
   render() {
-    const { id, labelText } = this.props;
+    const { id, labelText, className: customClassName, ...rest } = this.props;
```

Comment:
Quick note, I think the `className: customClassName` position only needs to happen if there is a conflicting `className` variable in scope. Otherwise, can keep it as `className`, I think 👍